File: Practices/time_of_day.py
#GLG period 7 time_of_day.py
hour1 = input("what time is it, in military time?") 

hour = int(hour1)

if hour >= 18 and hour <= 23 :
    print("Good Evening!")
elif hour >= 12 and hour <= 17:
    print("Good Afternoon!")
else:
    print("Good Nigth!")

File: C/variables.c/c/practises/time_of_day.c
// GLG 7th period time of day 
#include <stdio.h>

int main() {
    int hour;

    printf("What time is it, in military time? ");
    scanf("%d", &hour);

    if (hour >= 18 && hour <= 23) {
        printf("Good Evening!\n");
    }
    else if (hour >= 12 && hour <= 17) {
        printf("Good Afternoon!\n");
    }
    else {
        printf("Good Night!\n");
    }

    return 0;
}
